__tests__/PhotoAlbum.test.ts

```diff
@@ -0,0 +1,37 @@
+import { PhotoAlbum, Photo } from "./types";
+
+describe("PhotoAlbum class", () => {
+  it("adds new photo to photo album", () => {
```

Comment:
Test should tell what it is doing and with what.

```suggestion
  it("adds new photo with addPhoto", () => {
```